### DISCUSSION TITLE
Login System (Chapter 21)

### DISCUSSION THREAD

**Discussion by RenderWarrior** [PracticalBooks/Practical-Laravel#5] [General]
Hello! 
I'm learning a lot with your book!
But I noticed something different with the Laravel UI component.
In the book you told about changing the HOME attribute in app/Providers/RouteServiceProvider.php but it seems the latest Laravel UI changed the code... so you need to set the "/" url into different files. 
In your example in this repo I see you used into http/Controllers/Auth/LoginController.php 

protected $redirectTo = RouteServiceProvider::HOME;

at line 29, but actually the component has 

protected $redirectTo = '/home';

So you need to refactor all the files in the Controllers/Auth folder, changing the /home or implementing the old code.

**Comment by danielgara** [comment]
Hi @RenderWarrior, thanks for your update. You are completely right. The latest version of the Laravel UI component altered the initial setup and modified the internal code.

Thanks for your solution; it will help future readers. We are also planning to publish a second edition of our Laravel book later this year.

**Reply by RenderWarrior** [reply]
Oh great! I’ll buy it for sure! 😉
It would be great having more examples for improve the online store (if you’ll keep that example) for example a payment with paypal or stripe. Or improving the users privileges or adding a blog, or a slider with dynamic content 😉

**Reply by RenderWarrior** [reply]
And having some examples with Ajax calls would be great!
